posts/cka-preparation-2023.md

```diff
@@ -0,0 +1,87 @@
+---
+authors:
+- Iryna Chmelyk
+tags:
+- Kubernetes
+- CKA Certification
+- Cloud
+- DevOps
+date: 2023-12-12T11:29:50.000Z
+title: "CKA Certification Preparation: Why and How"
+image: https://github.com/iryna-chmelyk/blog-usa/blob/cka-preparation/images/2023/12/cka-certificate.png
+---
+
+![Certificate](https://github.com/iryna-chmelyk/blog-usa/blob/cka-preparation/images/2023/12/cka-certificate.png)
+A couple of months back I was searching for tips on how to prepare and pass the CKA (Certified Kubernetes Administrator) certification. I have looked through various materials, read quite a few discussions on this topic, and have chosen my training path. As a result, I was able to pass this exam on the first try. Now, I want to give back to the community of non-stop learners who have set their eyes on one of the "most wanted" certifications in the DevOps realm.
+
+# Relevance
+
+
+Kubernetes [(K8s)](https://kubernetes.io/) is the most widely used open source container orchestration platform on the market. Originally developed at Google, it is currently hosted by the Cloud Native Computing Foundation [(CNCF)](https://www.cncf.io/). Even though Kubernetes has been steadily gaining popularity for almost a decade, there are still not enough specialists with this skill set. More and more people are getting interested in this technology and Kubernetes-savvy engineers are in high demand. CNCF in collaboration with [The Linux Foundation](https://training.linuxfoundation.org/) has created Kubernetes Certification Programs to meet the needs of the market. What makes these exams so special besides the fact that they cover extremely sought-after knowledge areas, is that they take an entirely hands-on approach. There is no way to randomly guess your way into passing them. Not only do they certify theoretical knowledge, but they also validate practical experience working on K8s clusters.
+
+Depending on their career path, engineers can choose between three kinds of certifications:
+
+- [CKA](https://training.linuxfoundation.org/certification/certified-kubernetes-administrator-cka/) - Certified Kubernetes Administrator 
```

Comment:
Whole thing is really great... Even though blog is about CKA, it might be worth expanding on the 3 paths described and just giving a sentence or two of context to each as to why one might consider one of the alternatives.

Reply:
@lanefu I have added the descriptions of 3 certs. Please review.